docs/source/notebooks/bayesian_estimation_of_ode_parameters.ipynb

```diff
@@ -0,0 +1,591 @@
+{
```

Comment:
<p>It looks like there might be rendering issues here? I’m using reviewnb for this, so if there’s no issues on your side, we can let this comment go.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3578/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Hmm, that's weird.  I don't see those rendering issues when I open the notebook.  Maybe it's a thing with reviewnb.

Reply:
<p>The rendering issue seems to be resolved.</p>